src/components/common/searchbar.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react';
+import SearchIcon from "../../assets/search.svg";
+import { Image } from 'primereact/image';
+import { InputText } from "primereact/inputtext";
+
+type Props = {
+    showSearchButton: boolean;
+    openSearchField: boolean;
+    setOpenSearchField: (a: boolean) => void;
+    searchValue: string;
+    setSearchValue: (a: string) => void;
+};
+
+const SearchBar: FC<Props> = ({ showSearchButton, setOpenSearchField, openSearchField, searchValue, setSearchValue }) => {
+
+    return (
+        <div className="block">
+            <div className='flex'>
```

Comment:
Lines 17 and 18 can be merged in to a single div.

Reply:
This is not yet addressed.

Reply:
Merged them